src/Razor/src/Microsoft.VisualStudio.RazorExtension/source.extension.vsixmanifest

```diff
@@ -12,14 +12,15 @@
     <AllowClientRole>true</AllowClientRole>
   </Metadata>
   <Installation AllUsers="true" Experimental="true">
-    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.0,)" />
+    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.0,)">
+    </InstallationTarget>
   </Installation>
   <Dependencies>
     <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.7.2,)" />
   </Dependencies>
   <Prerequisites>
-    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[16.0,)" DisplayName="Visual Studio core editor" />
-    <Prerequisite Id="Microsoft.VisualStudio.Component.Roslyn.LanguageServices" Version="[16.0,)" DisplayName="Roslyn Language Services" />
+    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[17.0,)" DisplayName="Visual Studio core editor" />
```

Comment:
Shouldn't this be kept consistent with VS version?

Reply:
At first I was thinking so but I also thought about leaving it only to ensure that things don't actually work when you install this in a dev16 build 😄